pkg/util/httprequest/httprequest_test.go

```diff
@@ -25,20 +25,7 @@ func TestSchemeHost(t *testing.T) {
 				},
 			},
 			expectedScheme: "https",
-			expectedHost:   "example.com:443",
-		},
-		"X-Forwarded-Port overwrites X-Forwarded-Host port": {
```

Comment:
You can change the title to `"broken configuration"` and change the expected values instead of deleting the test.